docs/portal.md

```diff
@@ -32,22 +32,22 @@ At least one [shipment](references/corporate_accounts.html#shipment-resource) is
 #### Using an Allowance
 [Allowances](references/corporate_accounts.html#allowance-resource) provide a shortcut to determining the [modes](references/corporate_accounts.html#modes) and [units](references/corporate_accounts.html#unit-resource) necessary to create a shipment by grouping the units together. Allowances can be customized and this is set up for you during implementation. 
 #### Without an Allowance
-Select the [service(s)](references/corporate_accounts.html#modes) you want to be a part of this shipment and the [units](references/corporate_accounts.html#unit-resource) you wish to use as input to create a quote. Weight and volume units are required for all shipments, and for international FCL shipments you must also include a container size (20, 40, 40hc).
+Select the [service(s)](references/corporate_accounts.html#modes) you want to be a part of this shipment and the [units](references/corporate_accounts.html#unit-resource) you wish to use as input to get quoted prices. Weight and volume units are required for all shipments, and for international FCL shipments you must also include a container size (20, 40, 40hc).
 ## Pricing a Shipment
-In order to get the latest prices for a shipment, you first need to generate a [quote](references/corporate_accounts.html#quote-resource). Quotes are created at the bid stage. The current [price](references/corporate_accounts.thml#price-shipment) for a shipment at the [bid](references/corporate_accounts.html#status) stage is based on the most recent quote and the last [update](references/corporate_accounts.html#update-resource), for any stage passed bid it is based on the ordered service and attempting to create a quote will result in an error.
+The current [price](references/corporate_accounts.thml#price-shipment) for a shipment is based on the most recent  [update](references/corporate_accounts.html#update-resource). During the bid stage you will need to make a post prices request under certain conditions. The prices route will return a 404 and tell you when you need to post prices first.
```

Comment:
I think this should be a 400 not a 404, but I'm willing to be convinced otherwise. Looking through the 4xx status codes 400 or 404 seem to be the only real options to me.